+hw/+ptb/Window.m

```diff
@@ -385,6 +386,28 @@ function fillRect(obj, colour, rect)
     end
 
     function [nx, ny] = drawText(obj, text, x, y, colour, vSpacing, wrapAt)
+    % Calls PTB's `DrawFormattedText` to display text to the PTB Window
+    %
+    % Inputs:
+    %   `text`: the text to be displaye
+    %   `x`: a number, in pixels, of the x-position of the left border of 
```

Comment:
Your formatting is really inconsistent.  Try to stick to what's already in the repository code instead of coming up with a new style each time.  you switch between `` ` `` and `'`, `:` and `-`.

Reply:
ok, I'll make the formatting consistent